apps/web-staking/src/app/components/announcementBanner/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from 'react';
+import ExternalLinkIcon from "../../../../../../packages/ui/src/rebrand/icons/ExternalLinkIcon";
+import {CloseIcon} from "../../../../../../packages/ui/src/rebrand/icons/CloseIcon";
```

Comment:
@stanyyyyy This is not the correct way to import these files and it will break when building from the monorepo.

Please refactor this once the monorepo fix for the staking app is merged into develop